frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick_start_card_on_cluster_overview_ODC-4922.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Add quick starts card on cluster overview
+	As a user, I want to easily be able to access and discover Quick Starts from the Cluster Overview in administrator view.
+
+Background:
+    Given user is at administrator perspective
+    And user has created cr for sample-application-quickstart quick start
+    And user has created cr for install-associate-pipeline-quickstart quick start
+    And user has created cr for explore-pipeline-quickstart quick start
+    And user has created cr for add-healthchecks-quickstart quick start
+
+@regression
+Scenario: Quick Start card on Cluster Overview 
+   Given user is in administrator perspective
+   When user goes to Cluster Overview page
+   Then user can see quick tour card 
+   And user can see three Quick Start link present on it
```

Comment:
Rather than adding number, it will be great if you mention quick starts of which feature

Reply:
It depends on which quick start CR's added so the number will be more appropriate here